web-ui/src/main/resources/catalog/components/validationtools/GnmdInspireValidationDirective.js

```diff
@@ -47,13 +47,13 @@
                 scope.inspMdUuid = newValue;
 
                 gnConfigService.load().then(function(c) {
-                  // INSPIRE validator only support ISO19139 records.
-                  // TODO: For other schema support we may need to convert the record
-                  // to ISO19139 first. eg. ISO19115-3
+                  // INSPIRE validator only support ISO19139/115-3 records.
+                  // This assume that those schema have and ISO19139 formatter
+                  // which is the format supported by the validator
                   scope.isInspireValidationEnabled =
                     gnConfig[gnConfig.key.isInspireEnabled] &&
                     angular.isString(gnConfig['system.inspire.remotevalidation.url']) &&
-                    (gnCurrentEdit.schema === 'iso19139');
+                    gnCurrentEdit.schema.match(/iso19139|iso19115-3/) != null;
```

Comment:
Is gnCurrentEdit.schema always != null? otherwise, ".match" could be a problem.

Reply:
Should be https://github.com/geonetwork/core-geonetwork/blob/master/domain/src/main/java/org/fao/geonet/domain/MetadataDataInfo.java#L110